mdanalysissandbox/poem.py

```diff
@@ -0,0 +1,8 @@
+""" here is a docstring """
+
+def write_a_poem():
+    print("We must, in our research and courses\n
```

Comment:
I'd suggest to return a string instead of printing in a function.

Reply:
Addressed in 679c2e3557970220e71af008f396f3bd4c8eef91